docs(sources): add integrations directory with 31 source pages (ETL-980) (#739)
glassflow/clickhouse-etl: docs/app/sources/azure-event-hubs/page.mdx

---
title: 'Azure Event Hubs'
description: 'Stream events from Azure Event Hubs into ClickHouse with GlassFlow'
---
import { Callout } from 'nextra/components'

# Azure Event Hubs

Event Hubs is a large-scale event ingestion service by Azure designed for streaming millions of events per second. It is commonly used for telemetry pipelines, logging, and real-time analytics.

## Using Azure Event Hubs with GlassFlow

GlassFlow can ingest data from Azure Event Hubs into ClickHouse with the same pipeline capabilities available to all GlassFlow sources — deduplication, joins, filtering, and stateless transformations.

<Callout type="info">

> **Available on request.** Azure Event Hubs connectors are supported natively via the GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.

Reviewer: ```suggestion
**Available on request.** Azure Event Hubs connectors are natively supported in GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.
```
